server/src/domain/job/job.interface.ts

```diff
@@ -17,9 +17,15 @@ export interface IFaceThumbnailJob extends IAssetFaceJob {
   personId: string;
 }
 
+export interface IPersonJob extends IBaseJob {
+  personId: string;
+  ownerId: string;
```

Comment:
I don't think either of these are needed. You can just use `IEntityJob` and use id to pass the id of the entity (a person here)